lib/naxum/src/middleware/ack/maintain_progress.rs

```diff
@@ -27,27 +27,33 @@ impl MaintainProgressTask {
     }
 
     pub async fn run(mut self) {
-        trace!(si.naxum.task = Self::NAME, "running task");
-        debug!(si.naxum.task = Self::NAME, "first ack message");
+        trace!(
+            si.naxum.task = Self::NAME,
+            "running task (first ack message)"
+        );
+
         if let Err(err) = self.acker.ack_with(jetstream::AckKind::Progress).await {
             warn!(si.error.message = ?err, "failed initial ack");
         }
 
         loop {
             tokio::select! {
                 _ = self.shutdown_token.cancelled() => {
-                    info!(si.naxum.task = Self::NAME, "received cancellation");
+                    trace!(si.naxum.task = Self::NAME, "received cancellation for maintain progress task");
                     break;
                 }
                 _ = self.interval.tick() => {
-                    debug!(task = Self::NAME, "acking message with progress");
+                    trace!(task = Self::NAME, "acking message with progress");
                     if let Err(err) = self.acker.ack_with(jetstream::AckKind::Progress).await {
                         warn!(si.error.message = ?err, "failed to ack with progress");
                     }
                 }
             }
         }
 
-        info!(si.naxum.task = Self::NAME, "naxum shutdown complete");
+        trace!(
+            si.naxum.task = Self::NAME,
+            "naxum maintain progress shutdown complete"
+        );
```

Comment:
These are now all `trace` or `warn` for ease of use. The `trace` ones are very noisy and don't belong in debug runs